tests/middleware_tests/test_parameters_name_decoding_handler.py

```diff
@@ -19,24 +19,29 @@ def test_custom_options():
     """
     Test that default configuration is overrriden if custom configuration is provided
     """
-    options = ParametersNameDecodingHandlerOption(enable=False, characters_to_decode=[".", "-"])
+    options = ParametersNameDecodingHandlerOption(
+        enable=False, characters_to_decode=[".", "-"]
+    )
     handler = ParametersNameDecodingHandler(options)
 
     assert handler.options.enabled is not True
     assert "$" not in handler.options.characters_to_decode
     assert handler.options.get_key() == "ParametersNameDecodingHandlerOption"
 
+
 @pytest.mark.asyncio
 async def test_decodes_query_parameter_names_only():
     """
     Test that only query parameter names are decoded
     """
-    encoded_url = 'https://graph.microsoft.com?%24count=true&query=%24top&created%2din=2022-10-05&q=1%2b2&q2=M%26A&subject%2ename=%7eWelcome'
-    expected_url = 'https://graph.microsoft.com?$count=true&query=%24top&created-in=2022-10-05&q=1%2b2&q2=M%26A&subject.name=%7eWelcome'
+    encoded_url = "https://graph.microsoft.com?%24count=true&query=%24top&created%2din=2022-10-05&q=1%2b2&q2=M%26A&subject%2ename=%7eWelcome&%24empty="
+    expected_url = "https://graph.microsoft.com?$count=true&query=%24top&created-in=2022-10-05&q=1%2b2&q2=M%26A&subject.name=%7eWelcome&$empty="
```

Comment:
```suggestion
    encoded_url = "https://graph.microsoft.com?%24count=true&query=%24top&created%2din=2022-10-05&q=1%2b2&q2=M%26A&subject%2ename=%7eWelcome&%24empty"
    expected_url = "https://graph.microsoft.com?$count=true&query=%24top&created-in=2022-10-05&q=1%2b2&q2=M%26A&subject.name=%7eWelcome&$empty"
```

I think the test will fail with that change, but that's intended. This is the case I was referring to in my earlier comment and that I think we should support.

Reply:
Ah, i see. Pushed a change to address.